ProjectLighthouse.Servers.Website/Pages/Login/BannedUserPage.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@page "/banned"
+@using LBPUnion.ProjectLighthouse.Configuration
+@using LBPUnion.ProjectLighthouse.Localization.StringLists
+@model LBPUnion.ProjectLighthouse.Servers.Website.Pages.Login.BannedUserPage
+
+@{
+    Layout = "Layouts/BaseLayout";
+    Model.Title = Model.Translate(ModerationStrings.SuspensionHeading);
+
+    string timeZone = (string?)ViewData["TimeZone"] ?? TimeZoneInfo.Local.Id;
+    TimeZoneInfo timeZoneInfo = TimeZoneInfo.FindSystemTimeZoneById(timeZone);
+}
+
+<div class="ui middle aligned left aligned">
+    <h1>@Model.Translate(ModerationStrings.SuspensionHeading)</h1>  
+
+    <p>
+        @Model.Translate(ModerationStrings.SuspensionExplanation, ServerConfiguration.Instance.Customization.ServerName)
+    </p>         
+   
+    <p>
+        @if (Model.ModCase.ExpiresAt != null)
+        {
+            <span>
+                @Model.Translate(ModerationStrings.SuspensionExpiration, TimeZoneInfo.ConvertTime(Model.ModCase.ExpiresAt.Value, timeZoneInfo).ToString("M/d/yyyy @ h:mm tt"))
+            </span>
+        }
+        else
+        {
+            <span>
+                @Model.Translate(ModerationStrings.SuspensionExpiration, Model.Translate(ModerationStrings.DoesNotExpire))
+            </span>
+        } 
```

Comment:
The span tags should probably be outside the if statement so you don't have to write an opening and closing tag for each if body